docs/topics/templates/document-attributes.adoc

```diff
@@ -22,11 +22,11 @@ ifdef::mta[]
 :WebNameTitle: User Interface
 :WebNameURL: user_interface_guide
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.2.1
+:ProductVersion: 7.0,0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.2.1.GA-redhat-00002
-:ProductDistributionVersion: 6.2.1.GA-redhat
-:ProductDistribution: mta-6.2.1.GA-cli-offline.zip
+// :MavenProductVersion: 7.0.0.GA-redhat-00001
+:ProductDistributionVersion: 7.0.0.GA-redhat
+:ProductDistribution: mta-7.0.0.GA-cli-offline.zip
```

Comment:
I don't think we are distributing CLI like this . 

[linux-mta-cli.zip](http://download.lab.bos.redhat.com/rcm-guest/staging/jboss-migrationtoolkit/MTA-7.0.0/linux-mta-cli.zip)

Reply:
ProductDistribution is going to be redundant for MTA

ProductDistributionVersion needs two minor changes that i will do later 

See JIRA for details